saml/models/core/response.go

```diff
@@ -6,4 +6,53 @@ import (
 
 // Response is a SAML Response element.
 // See 3.3.3 http://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf
-type Response types.Response
+type Response struct {
+	types.Response
```

Comment:
I switched the approach here because I think it will be more flexible long term. For example, we can add additional fields to `Response` if needed or eventually migrate fields out of `types.Response`.